src/torchcodec/_core/Encoder.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <torch/types.h>
+#include "src/torchcodec/_core/FFMPEGCommon.h"
+
+namespace facebook::torchcodec {
+class Encoder {
+ public:
+  ~Encoder();
+
+  Encoder(const torch::Tensor wf, int sampleRate, std::string_view fileName);
+  void encode();
+
+ private:
+  void encode_inner_loop(
```

Comment:
Nit: camelCase instead of snake_case.

For the record, I prefer snake case, but I think being consistent is more important. If I could wave a magic wand and make our repo all snake case for variable and function names, I would. Class names should still be pascal case, I believe.

Reply:
I am more used to camel case too. If we really want to, surely there must exist some camelCase to smake_case auto converters?

Reply:
There probably are - I'm open to us doing a giant conversion at one point.